composer.json

```diff
@@ -28,7 +28,7 @@
     },
     "require": {
         "php": ">=7.0",
-        "nikic/php-parser": "^3.0.4",
+        "nikic/php-parser": "^3.0.5",
```

Comment:
Not if I want to require this version. I don't want to get the broken version because of caching.